src/Brick/Keybindings/Pretty.hs

```diff
@@ -124,7 +124,7 @@ mkKeybindEventHelp kc h =
           ByKey b ->
               (Comment "(non-customizable key)", [Verbatim $ ppBinding b])
           ByEvent ev ->
-              let name = fromJust $ keyEventName (keyConfigEvents kc) ev
+              let name = fromMaybe "<unnamed>" $ keyEventName (keyConfigEvents kc) ev
```

Comment:
@jtdaugherty I don't understand why this failed for me - I do not see `<unnamed>` in the [output](https://github.com/swarm-game/swarm/pull/1979#issuecomment-2195592963).

Reply:
I must have missed one event in `KeyEvents` and then fixed it before creating this PR.

I deleted one, and the text is shown now:
```markdown
| Keybinding | Event Name | Description |
| ---------- | ---------- | ----------- |
| `C-q` | (unnamed) | Open quit game dialog |
```